front/app/containers/Admin/pages/EditPageForm.tsx

```diff
@@ -81,16 +79,8 @@ const EditPageForm = ({ params: { pageId } }: Props & WithRouterProps) => {
     clHistory.push('/admin/pages');
   };
 
-  const renderFn = (pageId: string, code: TPageCode) => (
-    props: FormikProps<FormValues>
-  ) => {
-    return (
-      <PageForm
-        {...props}
-        pageId={pageId}
-        hideSlugInput={EDITABLE_SLUG_PAGES.has(code)}
-      />
-    );
+  const renderFn = (pageId: string) => (props: FormikProps<FormValues>) => {
+    return <PageForm {...props} pageId={pageId} hideSlugInput={false} />;
```

Comment:
I'm not as deep into this so not sure which pages will be editable here but: I'm not sure if the slug should be editable for all pages. I remember chats with Julienne about this... Some pages need a fixed URL, we have hard-coded URLs that depend on them being there. If you search the code base for `/pages/`

Reply:
I think it's like it was: EDITABLE_SLUG_PAGES = 'faq', 'about', 'custom'

Reply:
You are right, and this is already happening- all pages besides the ones with the codes 'faq', 'about' and 'custom' are filtered out on line 39. I changed to code now to make that more clear ('fixed' pages mean pages that cannot be added to navbar, and cannot have their slugs changed- this is defined in `services/pages`):

`const nonFixedPages = pages.filter(isNotFixed);`